### ISSUE TITLE
[Bug]: Bottles keep Loading (Intel)

### ISSUE THREAD

**Issue by oyren** [bottlesdevs/Bottles#1500]
### Describe the bug

Good day,

I am a bit confused here, since I got the problem on multiple machines with different distributions, and it seems like no one else has this problem.

So the problem as the title suggest is that I am stuck in the "Loading... Still loading, please wait." Window for like 3-20 minutes and then randomly Bottle is functional.
Before, I can't even click on the Menu button (three vertical dots).
And this is the case every start of Bottle.

So far, I have been able to observe this behavior under Arch KDE and Fedora 36 Gnome, both under X11 and Wayland. 
Mostly I use the Flatpak, but I also tested the Fedora Package.

Also, I tried to get a log, but if I start Bottles via shell it shows nothing.

Thanks

### To Reproduce

1. Install Bottle via Flatpak or Fedora dnf.
2. Run through welcome screen
3. Download some important stuff 
4. Experience ultra long loading time

### Package

Flatpak from Flathub

### Distribution

Arch, Fedora 36

### Health Check

```shell
Display:
    X.org: true
    X.org (port): :99.0
    Wayland: true
Graphics:
    vendors:
        intel:
            vendor: intel
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/intel_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/intel_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 5.17.9-arch1-1
Distro:
    Name: GNOME
    Version: '"42 (Flatpak runtime)"'
Disk:
    Total: 8256995328
    Free: 8256847872
RAM:
    MemTotal: 15.4GiB
    MemAvailable: 7.5GiB
Tools:
    cabextract: true
    p7zip: true
    patool: true
    glibc_min: '2.33'
Bottles_envs: null
```

### Display Protocol

Wayland

### Display Backend used by Bottles

I don't know

**Comment by mirkobrombin** [comment]
I've seen you use Intel, which is the only hardware I haven't tested for Bottles.

**Comment by mirkobrombin** [comment]
Added a small personal note to the title

**Comment by oyren** [comment]
Good catch, seems related, both notebooks I tested it so far have Intels inside. 
I tried it again with an AMD (NixOS Flatpak), and it works fine.

22.05.2022 12:37:25 Mirko Brombin ***@***.***>:

**Comment by mirkobrombin** [comment]
I'll keep this open but idk how to provide a fix :D

**Comment by TheEvilSkeleton** [comment]
@oyren can you create a new user, install Bottles from Flathub and test it on the new user? It might help to check if it's a user specific issue system-wide issue.

**Comment by ghtesting2020** [comment]
I am also using Intel. I wonder if this is related to my issue of Bottles crashing past the 2022.5.14-trento version
https://github.com/bottlesdevs/Bottles/issues/1476
can you try 2022.5.14-trento?
```
sudo flatpak update --commit=5381497ef2d46e49fb7380e71c2d89038d2df315b96b80a37d299e978884a12a com.usebottles.bottles
```

**Comment by oyren** [comment]
> I'll keep this open but idk how to provide a fix :D

Traceable.


> I am also using Intel. I wonder if this is related to my issue of Bottles crashing past the 2022.5.14-trento version #1476 can you try 2022.5.14-trento? In the ticket I made I am using Arch and another person who had issues is using Fedora. Just like you.
> 
> ```
> sudo flatpak update --commit=5381497ef2d46e49fb7380e71c2d89038d2df315b96b80a37d299e978884a12a com.usebottles.bottles
> ```
Nope seems like it has no effect.


> @oyren can you create a new user, install Bottles from Flathub and test it on the new user? It might help to check if it's a user specific issue.

I have already done that, but will try again to be on the safe side.

**Comment by oyren** [comment]
> > @oyren can you create a new user, install Bottles from Flathub and test it on the new user? It might help to check if it's a user specific issue.
> 
> I have already done that, but will try again to be on the safe side.

Still the same behavior.

**Comment by forest-hermit** [comment]
It happens to me also, but I am stuck in loading forever. Flatpak too. I couldn't use Bottles since weeks. It seems there is a problem with a directory.

Terminal output:

```
flatpak run Bottles
error: Invalid id Bottles: Names must contain at least 2 periods
john@computer:~$ flatpak run com.usebottles.bottles
11:38:49 (INFO) Catalog components loaded 
11:38:49 (INFO) Catalog installers loaded 
11:38:49 (INFO) Catalog dependencies loaded 
11:38:49 (INFO) Performing Bottles checks... 
11:38:49 (INFO) Dxvks found:
         - dxvk-1.10.1
         - dxvk-1.10
 
11:38:49 (INFO) Vkd3ds found:
         - vkd3d-proton-2.6
 
11:38:49 (INFO) Nvapis found:
         - dxvk-nvapi-v0.5.3
 
11:38:49 (INFO) Latencyflexs found:
         - latencyflex-v0.1.0
 
11:38:49 (INFO) Runners found:
         - sys-wine-7.0
         - caffe-7.7
         - GE-Proton7-20
 
11:38:50 (ERROR) Error while running async job: <function MainWindow.__on_start.<locals>.get_manager at 0x7f1963358940>
        Exception: [Errno 20] No es un directorio: '/home/john/.var/app/com.usebottles.bottles/data/bottles/bottles/system.reg/bottle.yml'
 
  File "/app/share/bottles/bottles/utils/threading.py", line 64, in __target
    result = self.task_func(*args, **kwargs)
  File "/app/share/bottles/bottles/window.py", line 279, in get_manager
    mng = Manager(window)
  File "/app/share/bottles/bottles/backend/managers/manager.py", line 118, in __init__
    self.checks(install_latest=False, first_run=True)
  File "/app/share/bottles/bottles/backend/managers/manager.py", line 135, in checks
    self.check_bottles()
  File "/app/share/bottles/bottles/backend/managers/manager.py", line 768, in check_bottles
    with open(_config, "r") as f:
Traceback (most recent call last):
  File "/app/share/bottles/bottles/window.py", line 222, in set_manager
    x for x in self.manager.runners_available if not x.startswith('sys-')
AttributeError: 'NoneType' object has no attribute 'runners_available
```

And after that last update this shows up at start:

```
[Errno 20] No es un directorio: '/home/USER/.var/app/com.usebottles.bottles/data/bottles/bottles/system.reg/bottle.yml'  File "/app/share/bottles/bottles/utils/threading.py", line 64, in __target
    result = self.task_func(*args, **kwargs)

  File "/app/share/bottles/bottles/window.py", line 279, in get_manager
    mng = Manager(window)

  File "/app/share/bottles/bottles/backend/managers/manager.py", line 118, in __init__
    self.checks(install_latest=False, first_run=True)

  File "/app/share/bottles/bottles/backend/managers/manager.py", line 135, in checks
    self.check_bottles()

  File "/app/share/bottles/bottles/backend/managers/manager.py", line 768, in check_bottles
    with open(_config, "r") as f:
```



Operating System: Kubuntu 22.04
KDE Plasma Version: 5.24.4
KDE Frameworks Version: 5.92.0
Qt Version: 5.15.3
Kernel Version: 5.15.0-37-generic (64-bit)
Graphics Platform: X11
Processors: 12 × Intel® Core™ i7-8750H CPU @ 2.20GHz
Memory: 31.2 GiB of RAM
Graphics Processor: NVIDIA GeForce GTX 1050 Ti/PCIe/SSE2

**Comment by oyren** [comment]
Have you done anything to see log messages, I just don't see any?
Btw I have a i7-8650u.

**Comment by forest-hermit** [comment]
> Have you done anything to see log messages, I just don't see any? Btw I have a i7-8650u.

For seeing terminal output: flatpak run com.usebottles.bottles

The other messages at start are displayed in a report window at start.

**Comment by oyren** [comment]
> flatpak run com.usebottles.bottles

Thanks, yes I did that, but the last times I only got the following:
``` bash
$ flatpak run com.usebottles.bottles

(bottles:2): Gtk-WARNING **: 11:51:14.019: Locale not supported by C library.
	Using the fallback 'C' locale.
```

Since the last update I seem to get logs again, maybe it can narrow down the problem:
``` bash
$ timedatectl && flatpak run com.usebottles.bottles
               Local time: Sun 2022-06-12 12:38:08 CEST
           Universal time: Sun 2022-06-12 10:38:08 UTC
                 RTC time: Sun 2022-06-12 10:38:08
                Time zone: Europe/Berlin (CEST, +0200)
System clock synchronized: no
              NTP service: inactive
          RTC in local TZ: no

(bottles:2): Gtk-WARNING **: 12:38:08.779: Locale not supported by C library.
	Using the fallback 'C' locale.
12:53:16 (INFO) Catalog components loaded
12:55:30 (INFO) Catalog installers loaded
12:57:39 (INFO) Catalog dependencies loaded
12:57:39 (INFO) Performing Bottles checks...
12:57:39 (INFO) Dxvks found:
	 - dxvk-1.10.1

12:57:39 (INFO) Vkd3ds found:
	 - vkd3d-proton-2.6

12:57:39 (INFO) Nvapis found:
	 - dxvk-nvapi-v0.5.3

12:57:39 (INFO) Latencyflexs found:
	 - latencyflex-v0.1.0

12:57:39 (INFO) Runners found:
	 - sys-wine-7.0
	 - caffe-7.5

12:57:45 (INFO) Bottles found:

``` 

Approx. 15 minutes until Bottles is usable.

**Comment by Lucki** [comment]
> Approx. 15 minutes until Bottles is usable.

I had this behavior sometimes in the past and traced it down to GitHub giving timeouts with ipv6 about 3 x 5min (=15min) until it tried on ipv4.
Could you run with `strace` and see at what step it hangs?

**Comment by lemonzest79** [comment]
I also had this issue when I started using an IPv6 Tunnel on my router, long story short setting my routers network MTU to 1400 fixed all my timeout/dead site issues including those with bottles taking an age to open.

**Comment by oyren** [comment]
> I had this behavior sometimes in the past and traced it down to GitHub giving timeouts with ipv6 about 3 x 5min (=15min) until it tried on ipv4.

Thanks to all, that was the cause.

**Comment by forest-hermit** [comment]
That's not my cause I think (after compare the terminal output), should I open a new issue?

**Comment by Lucki** [comment]
> Thanks to all, that was the cause.

AFAIK that should have been fixed by #932 but if it's still occurring then it should be open so it can get tracked?

**Comment by forest-hermit** [comment]
With the last update I no longer have the problem.
